src/app/utils/highlights.ts

```diff
@@ -69,6 +78,57 @@ async function getHighlightsAndTurnOn(classId: number) {
   }
 }
 
+async function postNewMatch(gameId: number): Promise<ActiveMatch> {
+  const username = 'peter123';
+  const data = JSON.stringify({ gameId: gameId, username: username });
+
+  try {
+    const response = await fetch(`${baseUrl}/matches`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: data,
+    });
+    const result = await response.json();
+
+    console.log('postNewMatch', result);
+    return result;
+  } catch (error) {
+    console.log(error);
+    return null;
+  }
+}
+
+async function postNewHighlight({
+  videoSrc,
+  type,
+  timestamp,
+}: MatchHighlight): Promise<ActiveMatch> {
+  if (!activeMatch) return null;
```

Comment:
I would add `activeMatch` as argument (or `matchId`). This allows to test and use this function in isolation.